src/main/java/org/jenkinsci/plugins/durabletask/FileMonitoringTask.java

```diff
@@ -58,19 +64,34 @@
 
     private static final String COOKIE = "JENKINS_SERVER_COOKIE";
 
+    /**
+     * Charset name to use for transcoding, or the empty string for node system default, or null for no transcoding.
```

Comment:
Wait, empty string and null have different handling? 

Uh..... 🐛 - if the behavior is different that should be a custom string, i.e. 'DEFAULT' or 'PRESERVE' or something, because otherwise this is going to trip us up somewhere.

Reply:
It is internal only, but sure a constant could be introduced for clarity.